Makefile

```diff
@@ -846,7 +845,7 @@ EF_BOARDS  := $(ALL_BOARDS)
 ifeq ($(UNAME), Linux)
 SIM_BOARDS := sim_posix_revolution
 else ifeq ($(UNAME), Darwin)
-SIM_BOARDS := sim_osx_revolution
+SIM_BOARDS := sim_posix_revolution
```

Comment:
I thought this would change to build the correct one.  Did I mess up?  will test a complete build later if able..